package.json

```diff
@@ -33,6 +33,7 @@
     "@storybook/addon-a11y": "^6.4.19",
     "@storybook/addon-actions": "^6.4.19",
     "@storybook/addon-essentials": "^6.4.19",
+    "@storybook/addon-knobs": "^6.4.0",
```

Comment:
We should migrate the `math-input` stories to `@storybook/addon-controls`, but I'll leave that for when we get Storybook going in this repo.